samigo/samigo-app/src/java/org/sakaiproject/tool/assessment/ui/listener/delivery/DeliveryActionListener.java

```diff
@@ -118,6 +119,7 @@
   private static final ResourceBundle eventLogMessages = ResourceBundle.getBundle("org.sakaiproject.tool.assessment.bundle.EventLogMessages");
   private static final ResourceLoader rb = new ResourceLoader("org.sakaiproject.tool.assessment.bundle.DeliveryMessages");
   private static final ResourceLoader ra = new ResourceLoader("org.sakaiproject.tool.assessment.bundle.AuthorMessages");
+  private String studentRichText = ServerConfigurationService.getString("samigo.studentRichText", "true");
```

Comment:
this should be a boolean.. also can you add documentation on this variable to config/default.sakai.properties

Reply:
Added!